charts/openshift-metering/templates/hive/hive-scripts-configmap.yaml

```diff
@@ -8,7 +8,13 @@ data:
     set -ex
     cp -v -L -r -f /hadoop-starting-config/* /hadoop-config/
     cd /hadoop-config/
-    faq '.configuration.property+=[{name: ("fs.azure.account.key." + env.AZURE_STORAGE_ACCOUNT_NAME + ".blob.core.windows.net"), value: env.AZURE_SECRET_ACCESS_KEY }]' core-site.xml > core-site.xml.temp
+
+
+    if test -d ../gcs-json; then
+      faq '.configuration.property+=[{name: ("fs.azure.account.key." + env.AZURE_STORAGE_ACCOUNT_NAME + ".blob.core.windows.net"), value: env.AZURE_SECRET_ACCESS_KEY } , {name: "fs.gs.auth.service.account.json.keyfile" , value: "/gcs-json/gcs-service-account.json" }]' core-site.xml > core-site.xml.temp
```

Comment:
Maybe we should try to not repeat the same logic for `faq` in two spots. Instead maybe you could call `faq` twice:
```
faq '.configuration.property+=[{name: ("fs.azure.account.key." + env.AZURE_STORAGE_ACCOUNT_NAME + ".blob.core.windows.net"), value: env.AZURE_SECRET_ACCESS_KEY }]' core-site.xml > core-site.xml.temp
if [ -a /gcs-json/gcs-service-account.json ]; then
   faq '.configuration.property+=[{name: "fs.gs.auth.service.account.json.keyfile" , value: "/gcs-json/gcs-service-account.json" }] >> core-site.xml.temp
fi
```